packages/snaps-cli/src/commands/manifest/implementation.test.ts

```diff
@@ -179,7 +179,7 @@ describe('manifest', () => {
             "chains": ["eip155:1", "eip155:2", "eip155:3"]
           }
         },
-        "platformVersion": "1.0.0",
+        "platformVersion": "6.13.0",
```

Comment:
This should be mocked so we don't need to update this for every release.

Reply:
Good point!